src/app/stacktracedialog.cpp

```diff
@@ -48,6 +48,8 @@ StacktraceDialog::~StacktraceDialog()
 
 void StacktraceDialog::setStacktraceString(const QString &sigName, const QString &trace)
 {
+    const QString gitHash = (strlen(QBT_GIT_HASH) > 0) ? QString {} : QString::fromUtf8("+git" QBT_GIT_HASH);
```

Comment:
`QString::fromLatin1()` is enough since hash should not contain non-latin1 characters.

Reply:
Also it seems should be `(strlen(QBT_GIT_HASH) > 0`.